docs/ros/ros/installation/robot.mdx

```diff
@@ -0,0 +1,127 @@
+---
+title: Robot installation
+sidebar_label: Robot
+sidebar_position: 1
+toc_min_heading_level: 2
+toc_max_heading_level: 4
+---
+
+### Operating System (OS) {#operating-system}
+
+ROS 2 Humble uses [Ubuntu 22.04](https://releases.ubuntu.com/jammy/) as its Tier 1 operating system. Though other operating systems are supported,
+it is highly recommended to use Ubuntu 22.04. For the robot PC, Ubuntu 22.04 Server should be installed. The server version of the OS will not install a
```

Comment:
What are the implications of installing the Desktop version rather than the Server version? Are there missing packages that the user needs to install manually? Or does Desktop actually prevent things from running?

Reply:
Desktop will just use more RAM, and is generally unnecessary on a PC that won't be connected to a monitor. Otherwise it will probably work fine.

Reply:
Ok, I'll let you decide whether or not we need a note to clarify that for customers.